src/6_branch.js

```diff
@@ -393,7 +393,9 @@ Branch.prototype['init'] = wrap(
 					"url": utils.getWindowLocation(),
 					"user_agent": navigator.userAgent,
 					"language": navigator.language,
-					"page_has_microdata": checkForMicroData()
+					"page_has_microdata": checkForMicroData(),
+					"screen_width": window.screen.width ? window.screen.width : 0,
+					"screen_height": window.screen.height ? window.screen.height : 0
```

Comment:
- Are you sure we want to default to 0? I would confirm with whoever will be using this data
- You can do `"screen_width": screen.width || 0,`, don't need to include window